src/FeatureSet.php

```diff
@@ -14,30 +14,31 @@
 
 namespace Ramsey\Uuid;
 
-use Ramsey\Uuid\Converter\TimeConverterInterface;
-use Ramsey\Uuid\Generator\PeclUuidTimeGenerator;
-use Ramsey\Uuid\Provider\Node\FallbackNodeProvider;
-use Ramsey\Uuid\Provider\Node\RandomNodeProvider;
-use Ramsey\Uuid\Provider\Node\SystemNodeProvider;
+use Ramsey\Uuid\Builder\DefaultUuidBuilder;
```

Comment:
Sorry, PHPCompanion for Sublime Text must've automatically sorted the use statements when it imported the `ValidatorInterface`